README.rst

```diff
@@ -659,6 +659,11 @@ Choose from the list of available fixers:
                         $strict param. Warning! This
                         could change code behavior.
 
+* **useless_return** [contrib]
```

Comment:
I suggest leaving this Fixer at `contrib`, we can always move it to Symfony level later when they settled the return null vs. return void issue.